### PR TITLE
Add layout transformation test

### PR DESCRIPTION
**Pull request by lukasm91**
Adds a performance test for Layout Transformation. No updates on plots layout (can be done after adding distributed boundary performance test).

### CODE REVIEW

**Comment by lukasm91** [comment]
launch strgrid

**Comment by lukasm91** [comment]
do perftest

**Comment by lukasm91** [comment]
do perftest

**Comment by lukasm91** [comment]
do perftest

**Comment by lukasm91** [comment]
do perftest

File: pyutils/perftest/stencils/structured.py [hunk 1]
```diff
@@ -10,36 +10,35 @@ def path(stencil):
 
 
 class Copy(Stencil):
-    stella_filter = 'BasicBenchmarks.copyStencilTest'
```
[the remark this answers is not in the document]
Comment (lukasm91) [reply pyutils/perftest/stencils/structured.py]: AFAIK there is no comparison with stella anymore? @fthaler advised me to remove this lines...

@fthaler Correct me if I'm wrong...

**Comment by lukasm91** [comment]
do perftest

File: pyutils/perftest/stencils/structured.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (havogt) [reply pyutils/perftest/stencils/structured.py]: ok, I think I remember something like this.

**Comment by havogt** [comment]
needs update of references

**Comment by lukasm91** [comment]
References updated again after last change (storages use halos -> data is not aligned anymore)

File: pyutils/perftest/stencils/structured.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (fthaler) [reply pyutils/perftest/stencils/structured.py]: That’s correct, STELLA comparison was removed in the last refactoring as it simplified a lot and was never used anyway…

**Comment by lukasm91** [comment]
> The icosahedral references are removed on purpose?

Restored out-dated references.

**Pull request merged**